### PR TITLE
Improve error message on ci setup fails

### PR DESCRIPTION
**Pull request by shenmo3**
## Description

For CI tests, if the setup or parameters errors out, the actual error message output can be confusing. This is due to `AfterSuite` running when some steps in `BeforeSuite` fails, either due to parameter error or setup issue. Because setup is not finished, `AfterSuite` clean up also encounters and throws error, causing confusion. This PR fixes two cases.

## Changes
1. Add checks for Nephe controller deployment and skip some checks if its not deployed.
2. Add checks for kubectl configuration and skip clean up if the cluster is not configured.
3. Fixed a formatting issue for namespace clean up.

### CODE REVIEW

File: test/integration/integration_suite_test.go [hunk 1]
```diff
@@ -212,19 +212,30 @@ var _ = AfterSuite(func(done Done) {
 		close(done)
 		return
 	}
+
+	if kubeCtl == nil {
+		logf.Log.Info("No cluster found")
```
Comment (reachjainrahul) [inline test/integration/integration_suite_test.go:217]: why is this info message. isnt this an error.

File: test/integration/integration_suite_test.go [hunk 2]
```diff
@@ -212,19 +212,30 @@ var _ = AfterSuite(func(done Done) {
...
 
 	By(cluster + ": Check for controllers' restarts")
 	err = utils.CheckRestart(kubeCtl)
 	if err != nil {
-		logf.Log.Error(err, "error restarting nephe controller")
-		controllersCored = &cluster
+		if apierrors.IsNotFound(err) {
+			logf.Log.Info("nephe controller not deployed")
+		} else {
+			logf.Log.Error(err, "error restarting nephe controller")
```
Comment (reachjainrahul) [inline test/integration/integration_suite_test.go:236]: if there is an error, should we return?

**Review by reachjainrahul — COMMENTED** [review]
Can we avoid AfterSuite, if BeforeSuite fails??

File: test/integration/integration_suite_test.go (hunk 2 above)
Comment (shenmo3) [reply test/integration/integration_suite_test.go:236]: Not really, this error indicates that the nephe controller has restarted during the test (pod restart count > 0). In this case we continue the AfterSuite clean up and collect support bundle or preserve setup later. At the end the test will error out.

File: test/integration/integration_suite_test.go (hunk 1 above)
Comment (shenmo3) [reply test/integration/integration_suite_test.go:217]: No this means kubectl is not configured, means setup is not complete, some error happened during BeforeSuite.

**Comment by shenmo3** [comment]
> Can we avoid AfterSuite, if BeforeSuite fails??

I think we should run AfterSuite even if BeforeSuite fails, because some setup might already be there and some are not due to partially ran BeforeSuite. AfterSuite should try its best to clean up so the cluster won't be in a partially configured confusing state.

**Review by reachjainrahul — DISMISSED** [review]
/LGTM

**Review by reachjainrahul — APPROVED** [review]
/LGTM

**Pull request merged**